### PR TITLE
Package rename

### PR DESCRIPTION
**Pull request by CuriousVini**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,31 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright © 2017 CDAP
```
Comment (anwar6953) [inline pom.xml]: 2019

**Review by anwar6953 — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (CuriousVini) [reply pom.xml]: thats because we did not have this license before. The plugin exists from 2017

Comment (anew) [reply pom.xml]: but you still need to update it now to 2017-2019, right?

**Review by anew — APPROVED** [review]

**Comment by CuriousVini** [comment]
squashing and merging.

**Pull request merged**